### DISCUSSION TITLE
Comparison and roadmap. What's the current status?

### DISCUSSION THREAD

**Discussion by Atan-D-RP4** [Automattic/harper#296] [General]
Since Harper identifies itself as an alternative to things like Grammarly in a free and open source way, I think it natural to ask what features from Grammarly, et al, will users have to give up on when moving over to Harper? Which, if any, of these missing features are in development?

I really like this project, since it was exactly what I was looking for to add to my Neovim coding/writing setup. And I feel it can be so much more powerful than it already is but don't know enough about the features on both ends to pinpoint how it go about it.

Some comparison of features would be nice.

**Comment by grantlemons** [comment]
`COMPARISON.md` was created relatively recently to show the differences between similar tools.
I will say, at the current moment Harper is not a fully-fledged grammar checker the likes of the other options as it is rather young, so it doesn't have as many lints as other tools.
That being said, Harper was just acquired by Automattic, so further development should speed up significantly.

All in all, it's fast right now, and a lot of the development has gone to core components, which makes it pretty easy to rapidly improve the actual linting, which should pick up soon.

**Reply by Atan-D-RP4** [reply]
I have seen COMPARISON.md and felt it to be too sparse in the info it provided. I would rather have a longer list of features that are or are not implemented by each of the projects considered in the comparison but I understand that that is a lot to ask.
